src/PrestaShopBundle/Resources/views/Admin/Improve/Design/positions.html.twig

```diff
@@ -165,11 +152,10 @@
 
                             <a class="btn btn-outline-primary btn-sm" href="{{ getAdminLink('AdminModulesPositions', true, linkParams) }}">
                               <i class="material-icons">edit</i>
-                              {{ 'Edit' | trans({}, 'Admin.Actions') }}
```

Comment:
Please keep this value but put in in the `aria-label` attribute of the `<a>`, like this:

```
<a ... aria-label="{{ 'Edit' | trans({}, 'Admin.Actions') }}"> ... </a>
```

Reply:
:ok_hand: